conda/common/configuration.py

```diff
@@ -1432,7 +1540,10 @@ def describe_parameter(self, parameter_name):
         if not isiterable(et):
             et = [et]
 
-        if isinstance(parameter._element_type, Parameter):
+        # If it doesn't have  a __name__ attribute it's an object not a class
+        has_name = all(hasattr(element, "__name__") for element in et)
```

Comment:
What about using `inspect.isclass` instead?

Reply:
I'll check that out. Thanks!